docs/getting-started/webhook.rst

```diff
@@ -5,9 +5,9 @@ Webhook component
 In order to provide advanced resource validation, cert-manager includes a
 ValidatingWebhookConfiguration_ resource which is deployed into the cluster.
 
-This allows cert-manager to validate that Issuer, ClusterIssuer and Certificate
-resources that are submitted to the apiserver are syntactically valid, and
-catch issues with your resources early on.
+This allows cert-manager to validate that Issuer, ClusterIssuer, Certificate,
+and CertificateRequest resources that are submitted to the apiserver are
```

Comment:
Given this list is still non-exhaustive (Order & Challenge resources too), we might want to just replace with "cert-manager API resources" or similar?